src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java

```diff
@@ -41,6 +41,7 @@ public String checkAndStoreFileAndUrlLog(HttpServletRequest request,
             indexName = storeLogs.getUniqueIndexName(indexName);
             final String response = storeFileAndUrlLogs(
                 request, indexName, fileContent, isUrl);
+            storeLogs.findErrorsInIndex(indexName);
```

Comment:
Have we tested that the errorIndex is formed?

Reply:
test added